examples/tv-app/linux/include/endpoint-configuration/config.ini

```diff
@@ -0,0 +1,41 @@
+[videoplayer]
+endpointId = 1
+
+[speaker]
+endpointId = 2
+
+[app3]
+endpointId=3
+vendorName=exampleVendorName1
+vendorId=1
```

Comment:
should we make spaces before/after = consistent?

Reply:
Updated.